app/models/manageiq/providers/redhat/infra_manager/inventory/strategies/v4.rb

```diff
@@ -128,7 +130,7 @@ class HostPreloadedAttributesDecorator < SimpleDelegator
       def initialize(host, connection)
         @obj = host
         @nics = connection.follow_link(host.nics)
-        @statistics = connection.follow_link(host.statistics)
+        @statistics = connection.follow_link(host.statistics) if host.statistics
```

Comment:
That is a bug in the oVirt engine, which has been already fixed:

restapi: Don't add host statistics by default
https://gerrit.ovirt.org/74925

The workaround is to check if `host.statistics` is a link before actually using it:

```ruby
@statistics = connection.link?(host.statistics) ? connection.follow_link(host.statistics) : host.statistics
```